clinica/pipelines/statistics_volume/statistics_volume_utils.py

```diff
@@ -161,13 +290,17 @@ def model_creation(tsv, contrast, idx_group1, idx_group2, file_list, template_fi
     return current_model, covariates
 
 
-def is_number(s: str):
-    """
+def is_number(s: str) -> bool:
+    """Returns True is the input can be converted to float, False otherwise.
 
-    Args:
-        s: (str) string to test if it can be converted into float
+    Parameters
+    ----------
+    s: str
+        String to test if it can be converted into float
 
-    Returns:
+    Returns
+    -------
+    bool
```

Comment:
When the docstring says "bool: True or False", it doesn't provide any useful information.
This function is very simple, I would totally remove the Parameters and Returns sections...